app/src/main/res/layout/fragment_activities.xml

```diff
@@ -1,21 +1,88 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/activities_fragment"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/sign_up_activity_background"
     tools:context=".view.ActivitiesFragment">
 
-    <com.google.android.material.floatingactionbutton.FloatingActionButton
-        android:id="@+id/create_activity_fab_btn"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="end|bottom"
-        android:src="@drawable/ic_add"
-        android:foregroundTint="@color/white"
-        android:backgroundTint="@color/dark_blue"
-        android:layout_margin="16dp"
-        android:contentDescription="@string/create_activity" />
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <ImageView
```

Comment:
Placing our logo here does not make sense to me, screen space is already limited on phones and we add the logo here to block a quarter of the screen! I would just remove it.
I would also place the "create activity" button at the bottom of the screen to make it easily reachable

Reply:
You got it